react/IntentHeader/index.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styles from './styles'
+
+export const IntentHeader = ({ appName, appEditor, appIcon }) =>
+  (
+    <h2 className={styles['intentHeader']}>
+      <div className={styles['intentHeader-title']}>
+        <img className={styles['intentHeader-icon']} src={appIcon} />
+        {appEditor && <span>{appEditor}&nbsp;</span> }
+        {appName}
+      </div>
```

Comment:
Why an extra div ?

Reply:
Shouldn't the icon and the text be vertically aligned ? I would use a Media here.

Reply:
Now that you pointed that out, actually the `h2` & `div` are inverted. It should be a wrapping `div` and `h2` as a child that wrap the app name, and now that it it an iframe, `h2` should be an `h1`.
Also, the need for an extra wrapper is more because some other stuff may (or may not, not quite validated yet) come in this header that wouldn't be a title.

If I remember correctly `<Media />` doesn't allow to choose the markup, right?